.github/workflows/testsuite.yml

```diff
@@ -16,81 +16,94 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v4
+
     - name: Cache dependencies
-      uses: actions/cache@v2
+      uses: actions/cache@v4
       with:
         path: ~/.m2/repository
         key: ${{ runner.os }}-clj-${{ hashFiles('**/project.clj') }}
         restore-keys: |
           ${{ runner.os }}-clj-
+
     - name: Setup Java ${{ matrix.jdk }}
-      uses: actions/setup-java@v1.4.3
+      uses: actions/setup-java@v4
       with:
+        distribution: temurin
         java-version: ${{ matrix.jdk }}
+
     - name: Setup Clojure
-      uses: DeLaGuardo/setup-clojure@3.1
+      uses: DeLaGuardo/setup-clojure@13.1
       with:
         lein: 2.9.5
-    - name: Install dependencies
-      run: |
-        npm ci
+
     - name: Run tests
       run: ./scripts/test.sh clj
 
   build-cljs:
     name: ClojureScript
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
+
       - name: Cache dependencies
         uses: actions/cache@v2
         with:
           path: |
             ~/.m2/repository
-            **/node_modules
-          key: ${{ runner.os }}-cljs-${{ hashFiles('**/project.clj', '**/package-lock.json') }}
+          key: ${{ runner.os }}-cljs-${{ hashFiles('**/project.clj') }}
```

Comment:
why not cache node_modules?

Reply:
setup-node task takes care of that already